Documentation/advanced-configuration.md

```diff
@@ -0,0 +1,318 @@
+# Advanced Cluster Configuration
+
+These examples show how to perform advanced configuration tasks on your Rook storage cluster.
+
+## Prerequisites
+Most of the examples make use of the `ceph` client command.  A quick way to use the Ceph client suite is from a [Rook Toolbox](https://github.com/rook/rook/blob/master/Documentation/toolbox.md) container.
+
+The Kubernetes based examples assume Rook OSD pods are in the `rook` namespace.  If you run them in a different namespace, modify `kubectl -n rook [...]` to fit your situation.
+
+## Reweight your OSDs
+If your storage nodes have varying sizes of storage drives, you may benefit from changing their [CRUSH Map](http://docs.ceph.com/docs/kraken/rados/operations/crush-map/) Weight.  This setting controls the ratio of data that should be distributed to each device.  Right now newly provisioned storage devices on a single node get the same weight.
+
+A useful view of the CRUSH Map is generated with the following command:
+```bash
+ceph osd tree
+```
+
+### Storage capacity strategy
+It is common to set the weight for an OSD relative to the storage capacity of its drive in TiB, for example a 500GiB drive would get a weight of `.5`.  When all OSDs follow this convention, data will be distributed in such a way that avoids filling-up smaller drives before larger ones.  This will be the default behavior on new Rook clusters once [#477](https://github.com/rook/rook/issues/477) is resolved.
+
+#### Gathering OSD details
+First, if you're unsure which drives are managed by which OSDs then the following commands will help.
+
+##### Standalone
+```bash
+# Run this on each storage node
+# Tested with bluestore devices only
+
+echo "Node:\t" $(hostname -s)
+for i in /var/lib/rook/osd*; do echo "${i} $(lsblk -n -o NAME,SIZE $(readlink -f ${i}/block))"; done|sort -V
+```
+
+##### Kubernetes
+```bash
```

Comment:
That's a keyword for markdown syntax highlighting.  Looked better than no keyword when rendered, I think.